### DISCUSSION TITLE
CKAN latest: difficult to set up a web frontend with SSL support

### DISCUSSION THREAD

**Discussion by thebe14** [ckan/ckan#6395] [Show and tell]
I would like to install the latest version of CKAN but limit access to it via HTTPS only. This is very difficult to do properly, even using the [wiki page about web frontends](https://github.com/ckan/ckan/wiki/CKAN-2.9---New-Web-Server-options). Ideally, the installation guide for installing CKAN using Docker compose should install it with SSL support, including automatically requested/renewed Let's Encrypt SSL certificates.

I went through the trial-and-error to make this work, will submit a PR for it soon.

**Comment by kitsunemax** [comment]
nice @thebe14  ... will be great to have an update for proper Let's Encrypt and auto-renew on install with docker  :)
